src/AltinnCore/Common/Helpers/InstantiationHelper.cs

```diff
@@ -55,7 +55,7 @@ public static List<Party> FilterPartiesByAllowedPartyTypes(List<Party> parties,
                 else if (!IsPartyAllowedToInstantiate(party, partyTypesAllowed) && isChildPartyAllowed)
                 {
                     party.ChildParties = new List<Party>();
-                    party.OnlyHiearhyElementWithNoAccess = true;
+                    party.OnlyHiearchyElementWithNoAccess = true;
```

Comment:
again typo mistake, it should be "hierarchy" :)

Reply:
Onit :)